core/domain/email_manager_test.py

```diff
@@ -88,6 +90,198 @@ def test_sender_id_validation(self):
         # pylint: enable=protected-access
 
 
+class ExplorationMemebershipEmailTests(test_utils.GenericTestBase):
+    """Tests that sending exploration membership email works as expected."""
+
+    def setUp(self):
+        super(ExplorationMemebershipEmailTests, self).setUp()
+
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.new_user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+        self.exploration = self.save_new_default_exploration(
+            'A', self.editor_id, 'Title')
+
+        self.email_subject_tempate = (
```

Comment:
Actually, no -- when I said a "golden email", I meant just a raw string to compare with -- no interpolation, no calculations. Basically, there's an EXPECTED_EMAIL_SUBJECT and an EXPECTED_EMAIL_HTML_BODY and an EXPECTED_EMAIL_PLAINTEXT_BODY -- and that's it. See e.g. the stuff in SignupEmailTests.

Otherwise, you're testing implementation rather than interface (it should be the opposite).